libethereum/Account.cpp

```diff
@@ -88,11 +90,12 @@ namespace
 	string const c_balance = "balance";
 	string const c_nonce = "nonce";
 	string const c_code = "code";
+	string const c_file = "file";  ///< A file containg a code as bytes.
```

Comment:
I think `file` may be misleading to mean loading an account JSON from a file, e.g`{accounts: { "0x0044...": { file: "/path" } } }`

Reply:
Back to "codeInFile" or "codeFromFile"?

Reply:
I don't mind, probably the second is more readable.